src/main/java/obelix/ObelixCache.java

```diff
@@ -110,35 +101,25 @@ public void run() {
                 LOGGER.info("We're going to build the cache for all users!");
 
                 int usersAdded = 0;
-                try (Transaction tx = graphDb.beginTx()) {
-                    for (String userid : this.userGraph.getAll()) {
-                        redisQueueManager.push(new ObelixQueueElement("user_id", userid));
-                        usersAdded += 1;
-                    }
-                    tx.success();
+                for (String userid : this.userGraph.getAllUsers()) {
```

Comment:
Since we're refactoring.. I think that the getAllUsers should return User-objects, not strings